docker/aws_hub.properties

```diff
@@ -55,8 +55,8 @@ app.runNtpMonitor=false
 #This is the time of the oldest live item in a hub.  With the exception of historical, we know not to query before
 # this time.
 #app.birthDay=2015/01/01
-# Enable DataDog metrics
-data_dog.enable=false
+# Enable metrics services
+metrics.enable=false
```

Comment:
I'm unfamiliar with how we handle all our config management. Are we going to need to loop in Joe to get these changes pushed out or do they get copied as part of our build + deploy pipeline already?

Reply:
We will need to loop Joe in I believe, these are just for example

Reply:
If you have write permissions (which you should) you can update the configs for each environment within the salt-states BitBucket repo:

https://bitbucket.flightglobal.io/projects/PDX-SRE/repos/salt-states